geps/gep-1731/index.md

```diff
@@ -304,12 +304,12 @@ type HTTPRouteRetry struct {
     // route, the entire duration of the initial request and any retry attempts
     // MUST not exceed the Request timeout duration. If any retry attempts are
     // still in progress when the Request timeout duration has been reached,
-    // these SHOULD be cancelled if possible and the Gateway MUST immediately
+    // these SHOULD be canceled if possible and the Gateway MUST immediately
```

Comment:
For any other readers curious, apparently upstream Kubernetes has decided that English language documentation will be written in US English, so this and the related changes appear to be correct. https://kubernetes.io/docs/contribute/style/style-guide/#language

Reply:
Yea the linter caught it in docs, so I went back and updated the GEP too.